typescript-operators-and-expressions/main.ts

```diff
@@ -0,0 +1,38 @@
+const width: number = 5;
+const len: number = 10;
+const area: number = width * len;
+console.log('value of area: ', area);
+console.log('type of area: ', typeof area);
+
+const bill: number = 200;
+const payment: number = 1000;
+const change: number = payment - bill;
+console.log('value of change: ', change);
+console.log('type of change: ', typeof change);
+
+const quizzes: number = 90;
+const midterm: number = 100;
+const final: number = 40;
+const grade: number = (quizzes + midterm + final) / 3;
+console.log('value of grade: ', grade);
+console.log('type of grade: ', typeof grade);
+
+const firstName: string = 'Liam';
+const lastName: string = ' Nguyen';
+const fullName: string = firstName + lastName;
```

Comment:
Don't forget to add a space! Your first and last name should be separated with a space